azure-resources/Web/sites/recommendations.yaml

```diff
@@ -113,3 +113,63 @@
   learnMoreLink:
     - name: Configure web apps in Azure App Service
       url: "https://learn.microsoft.com/azure/app-service-web/web-sites-configure"
+
+- description: Health check
```

Comment:
This should be worded in the form of a recommendation

Reply:
Reworded to Enable Health check for App Services.